locales/en.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    HomeHeaderRight: {
```

Comment:
I suggest to use Scene/Component + `_` + key

HomeHeaderRight_Title
Profile_Follow_Title

something like this